### PR TITLE
Kafka config use in the tutorial (no Service Discovery used)

### PR DESCRIPTION
**Pull request by ignasi35**

### CODE REVIEW

File: docs-source/docs/modules/shopping-microservices/examples/shopping-cart-service-scala/src/main/resources/kafka.conf [hunk 1]
```diff
@@ -2,9 +2,7 @@ shopping-cart-service {
 
   kafka.topic = "shopping_cart_events"
 
-  kafka.producer: ${akka.kafka.producer} {
-    service-name = "shopping-kafka-broker"
-  }
+  kafka.producer: ${akka.kafka.producer}
```
Comment (patriknw) [inline docs-source/docs/modules/shopping-microservices/examples/shopping-cart-service-scala/src/main/resources/kafka.conf:5]: Without the `{` I find that syntax strange. Include empty `{}` or
```
kafka.producer = ${akka.kafka.producer}
```

Comment (ignasi35) [reply docs-source/docs/modules/shopping-microservices/examples/shopping-cart-service-scala/src/main/resources/kafka.conf:5]: You're right, I missed the `:`. Using the `=` is the cleanest.

**Review by ennru — COMMENTED** [review]
Alternatively, don't introduce `kafka.producer` and just use the default `akka.kafka.producer` from `ProducerSettings.ConfigPath` instead.

**Comment by patriknw** [comment]
> Alternatively, don't introduce kafka.producer

Right, that would only be needed if having to connect to two or more Kafka brokers?

**Comment by ennru** [comment]
> Right, that would only be needed if having to connect to two or more Kafka brokers?

Only if you want to pass different settings from `application.conf` to more than one producer. The broker addresses are most often passed in code (`withBootstrapServers`).

**Comment by patriknw** [comment]
ok, [those settings](https://github.com/akka/alpakka-kafka/blob/master/core/src/main/resources/reference.conf#L6) look rather advanced and there is no obvious reason why different settings would be needed for different producers. Let's simplify and use `akka.kafka.producer` directly instead.

**Review by patriknw — APPROVED** [review]
LGTM, nice

**Comment by ignasi35** [comment]
Moving out of the complexity and into defaults made it all cleaner.

**Review by ennru — APPROVED** [review]

**Pull request merged**